app/src/main/java/it/niedermann/nextcloud/deck/persistence/sync/SyncManager.java

```diff
@@ -1378,6 +1379,23 @@ public void deleteSingleCardWidgetModel(int widgetId) {
         doAsync(() -> dataBaseAdapter.deleteSingleCardWidget(widgetId));
     }
 
+    public void addStackWidget(int appWidgetId, long accountId, long stackId, boolean darkTheme) {
+        doAsync(() -> dataBaseAdapter.createStackWidget(appWidgetId, accountId, stackId, darkTheme));
+    }
+
+    @WorkerThread
+    public StackWidgetModel getStackWidgetModelDirectly(int appWidgetId) throws NoSuchElementException {
+        final StackWidgetModel model = dataBaseAdapter.getStackWidgetModel(appWidgetId);
```

Comment:
Please rename ` dataBaseAdapter.getStackWidgetModel()` to ` dataBaseAdapter.getStackWidgetModelDirectly()`, just for clarity